### PR TITLE
CLOUDP-137158: [Atlas CLI] Add support for remaining Processes APIs

### PR DESCRIPTION
**Pull request by gssbzn**
## Proposed changes

_Jira ticket:_ CLOUDP-137158

## Checklist

- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [x] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [x] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [x] I have run `make fmt` and formatted my code

## Further comments

```bash
$ atlas processes describe atlas-l71pkp-shard-00-00.vnp8x.mongodb-dev.net:27017

ID                                                     REPLICA SET NAME       SHARD NAME   VERSION
atlas-l71pkp-shard-00-00.vnp8x.mongodb-dev.net:27017   atlas-l71pkp-shard-0                5.0.12
```

### CODE REVIEW

File: internal/cli/atlas/metrics/databases/describe.go [hunk 1]
```diff
@@ -65,8 +65,9 @@ func DescribeBuilder() *cobra.Command {
 	const argsN = 2
 	opts := &DescribeOpts{}
 	cmd := &cobra.Command{
-		Use:   "describe <host:port> <databaseName>",
-		Long:  `To retrieve the hostname and port needed for this command, run mongocli atlas process list.`,
+		Use: "describe <host:port> <databaseName>",
+		Long: fmt.Sprintf(`To retrieve the hostname and port needed for this command, run:
```
Comment (gssbzn) [inline internal/cli/atlas/metrics/databases/describe.go:69]: fix for hardcoded entrypoint

File: internal/cli/default_setter_opts.go [hunk 2]
```diff
@@ -38,7 +38,7 @@ type ProjectOrgsLister interface {
 	Projects(*atlas.ListOptions) (interface{}, error)
 	Organization(id string) (*atlas.Organization, error)
 	Organizations(*atlas.OrganizationsListOptions) (*atlas.Organizations, error)
-	GetOrgProjects(string, *atlas.ListOptions) (interface{}, error)
+	GetOrgProjects(string, *atlas.ProjectsListOptions) (interface{}, error)
```
Comment (gssbzn) [inline internal/cli/default_setter_opts.go:41]: updating the libs introduced this change

File: internal/store/api_keys_access_list.go [hunk 3]
```diff
@@ -36,18 +36,6 @@ type OrganizationAPIKeyAccessListCreator interface {
 	CreateOrganizationAPIKeyAccessList(string, string, []*atlas.AccessListAPIKeysReq) (*atlas.AccessListAPIKeys, error)
 }
 
-type OrganizationAPIKeyWhitelistLister interface {
```
Comment (gssbzn) [inline internal/store/api_keys_access_list.go:39]: WhitelistAPIKeys no longer exists

File: internal/cli/atlas/processes/describe.go [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+func (opts *DescribeOpts) Run() error {
+	r, err := opts.store.Process(opts.ConfigProjectID(), opts.host, opts.port)
+	if err != nil {
+		return err
+	}
+
+	return opts.Print(r)
+}
+
+// DescribeBuilder atlas process(es) list --projectId projectId.
```
Comment (blva) [inline internal/cli/atlas/processes/describe.go:58]:
```suggestion
// DescribeBuilder atlas process(es) describe <hostname:port> --projectId projectId.
```

File: internal/cli/atlas/processes/describe.go [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+func DescribeBuilder() *cobra.Command {
+	opts := &DescribeOpts{}
+	cmd := &cobra.Command{
+		Use:   "describe <hostname:port>",
```
Comment (blva) [inline internal/cli/atlas/processes/describe.go:62]: can we pelase add an `Example` to the docs

**Review by blva — CHANGES_REQUESTED** [review]

File: internal/cli/atlas/processes/describe.go (hunk 5 above)
Comment (gssbzn) [reply internal/cli/atlas/processes/describe.go]: done

**Review by blva — DISMISSED** [review]
LGTM

File: docs/mongocli/command/mongocli-atlas-processes-describe.txt [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+List MongoDB processes for your project.
```
Comment (sarahsimpers) [inline docs/mongocli/command/mongocli-atlas-processes-describe.txt:15]:
```suggestion
Return the details for the MongoDB process you specify.
```

**Review by sarahsimpers — CHANGES_REQUESTED** [review]
Question/change for the description

**Review by sarahsimpers — APPROVED** [review]
LGTM

**Pull request merged**